packages/portal/src/components/search/SideFilters.vue

```diff
@@ -77,17 +77,49 @@
                   :api-options="apiOptions"
                   @changed="changeFacet"
                 />
-                <SideSwitchFilter
-                  v-if="contentTierFacetSwitch"
-                  :value="filters.contentTier"
-                  name="contentTier"
-                  :label="$t('facets.contentTier.options.0')"
-                  checked-value="&quot;0&quot;"
-                  :unchecked-value="null"
-                  :default-value="null"
-                  :collection="collection"
-                  @changed="changeFacet"
-                />
+                <b-button
```

Comment:
Add a `aria-controls` aria-expanded` attributes to this button.

Reply:
Done.